components/dashboard/src/login/SetupPending.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+import { Heading2, Subheading } from "../components/typography/headings";
+import cubicleImg from "../images/cubicle.png";
+import cubicleImg2x from "../images/cubicle-2x.png";
+import gitpodIcon from "../icons/gitpod.svg";
+import classNames from "classnames";
+
+type Props = {
+    alwaysShowHeader: boolean;
+};
+export const SetupPending: FC<Props> = ({ alwaysShowHeader }) => {
+    return (
+        <div className="flex-grow flex items-center justify-center p-4">
+            <div className={classNames(alwaysShowHeader ? "" : "lg:hidden", "absolute top-0 left-0 right-0")}>
+                <div className="flex items-center justify-center items-center py-3 space-x-1">
+                    <img src={gitpodIcon} className="w-6 h-6" alt="Gitpod's logo" />
+                    <span className="text-lg">Gitpod</span>
```

Comment:
**issue(non-blocking)**: Noticed this from the previous PRs related to the admin onboarding, but could we use the logo with the wordmark here? However, in this case, we would also have to use two different logo versions for the dark and light theme.

<img width="1906" alt="Frame 1470" src="https://github.com/gitpod-io/gitpod/assets/120486/84cc556b-7d95-4c32-96f0-a9c8170f3fba">